test/unit/test_deep_learning_containers.py

```diff
@@ -19,7 +19,7 @@
 import pytest
 
 
-from sagemaker_pytorch_container import deep_learning_container as deep_learning_container_to_test
```

Comment:
I think instead of placing the deep_learning_container.py in the docker/ folder which points to the framework dockerfiles according to versions, it's better to create a new folder (serving a different purpose that src/ which hosts SM-specific scripts) and place the deep_learning_container.py there, other common utils that might be added could go in there as well in the future. We can abstract that folder as a hosted package later.